### PR TITLE
Add coloring for $ and wordSepartors 

### LINKED ISSUE #2546: Select $variable with double click in editor
As a developer I want to select a variable including the $ so that I don't have to select it manually by sliding.

When a user double-click a variable, the whole variable including the $ is selected

### PR DESCRIPTION
**Pull request by MJECloud**
## PR Summary

Adding scopes in the theme to make the $ orange aswell. 
image

Also adding wordSepartors to ISE Mode 
Fixes #2546

## PR Checklist

Note: Tick the boxes below that apply to this pull request by putting an `x` between the square brackets.
Please mark anything not applicable to this PR `NA`.

- [x] PR has a meaningful title
- [x] Summarized changes
- [ ] PR has tests
- [x] This PR is ready to merge and is not work in progress
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready

### CODE REVIEW

File: src/features/ISECompatibility.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export class ISECompatibilityFeature implements IFeature {
         { path: "powershell.integratedConsole", name: "focusConsoleOnExecute", value: false },
         { path: "files", name: "defaultLanguage", value: "powershell" },
         { path: "workbench", name: "colorTheme", value: "PowerShell ISE" },
+        { path: "editor", name: "wordSeparators", value: "`~!@#%^&*()-=+[{]}\\|;:'\",.<>/?" }
```
Comment (rjmholt) [inline src/features/ISECompatibility.ts:27]: Did this value for the word separators come from anywhere in particular? I'm wondering about the following: `~`, `@`, `#`, `^`, `*`, `-`, `\`, `:`.

It depends on the context in which I think of where we're defining "words" too. In expression mode, `*` and `-` are word separators, but in command mode they're not. I suspect we should prefer expression mode, but want to raise it in any case

Comment (TylerLeonhardt) [reply src/features/ISECompatibility.ts:27]: It's the default minus the `$` - I originally said this here:
https://github.com/PowerShell/vscode-powershell/issues/2546#issuecomment-597328583

Not perfect... but better.

Comment (MJECloud) [reply src/features/ISECompatibility.ts:27]: Right, i used the example Tyler mentioned in the comment.
Is there a better implementation?

**Comment by TylerLeonhardt** [comment]
@MJECloud looks like there's a conflict now with a PR that just went in. Can you rebase or resolve the conflict?

File: src/features/ISECompatibility.ts (hunk 1 above)
Comment (rjmholt) [reply src/features/ISECompatibility.ts:27]: I'm happy with better. We can improve it in future as needed

**Comment by MJECloud** [comment]
Resolved the conflict.

**Comment by TylerLeonhardt** [comment]
Thanks for doing that!

**Pull request merged**